Felienne/hedy
# Improves program time-out handling

**Description**
In this PR we improve the program time-out by making it more dependent on the current level. As users are unable to create very large program from levels 1 till 6 as we don't support loops as the line limit is `100` lines we should take this into account with the time-out. In this PR we remove the time-out for levels 1 till 6 and alter the values for the higher levels.

**Fixes**
This PR fixes #2554.

**How to test**
Running the following program on level 2:

```
print Let's go to sleep!
sleep 30
print Wake up!
```

Then run the following program on level 8:
```
print 'knock knock'
sleep 30
print 'knock knock!'
print 'Who is there?'
```

Verify that the first case won't terminate on a time-out whereas the second one will.
